forge-gui/res/cardsfolder/upcoming/darigaaz_shivan_champion.txt

```diff
@@ -0,0 +1,13 @@
+Name:Daringaaz, Shivan Dragon
+ManaCost:2 B R G
+Types:Legendary Creature Dragon
+PT:5/5
+K:Flying
+T:Mode$ Phase | Phase$ End of Turn | Execute$ TrigDraft | ValidPlayer$ You | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your end step, conjure a random card from CARDNAME's spellbook into exile face down with three egg counters on it. It gains "at the beginning of your upkeep, if this card is exiled, remove an egg counter from it. Then if it has no egg counters on it, put it onto the battlefield.  
+SVar:TrigDraft:DB$ MakeCard | Conjure$ True | AtRandom$ True | Spellbook$ Shivan Dragon,Moonveil Regent,Terror of the Peaks,Leyline Tyrant,Immersturm Predator,Manaform Hellkite,Bone Dragon,Demanding Dragon,Skarrgan Hellkite, Thunderbreak Regent,Black Dragon,Skyship Stalker,Red Dragon | WithCountersType$ EGG | WithCountersAmount$ 3 | Zone$ Exile | ExileFaceDown$ True | RememberDrafted$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ RemoveEggCounter | Duration$ Permanent | Name$ Daringaaz, Shivan Dragon's Perpetual Effect
```

Comment:
I think this should be an Animate, not an Effect

Reply:
Good point, though generally the cardscript is very rough. I'm having trouble adding the param for counters when you exile the made card